app/controllers/concerns/phone_confirmation_flow.rb

```diff
@@ -65,7 +72,26 @@ def send_confirmation_code
     # user's session. Re-sending the confirmation code doesn't generate a new one.
     self.confirmation_code = generate_confirmation_code unless confirmation_code
 
-    SmsSenderOtpJob.perform_later(confirmation_code, unconfirmed_phone)
+    if current_otp_delivery_method == :voice
```

Comment:
Added specs to cover this: https://github.com/18F/identity-idp/blob/46fac41d7ac76871e97076597463d19f4fb68631/spec/controllers/devise/two_factor_authentication_controller_spec.rb#L340-L351 and https://github.com/18F/identity-idp/blob/46fac41d7ac76871e97076597463d19f4fb68631/spec/controllers/devise/two_factor_authentication_controller_spec.rb#L419-L434